### ISSUE TITLE
digital optical audio out doesn't work with sound visualization

### ISSUE THREAD

**Issue by sam-mckay** [psieg/Lightpack#145]
Using a optical cable from my pc to a home cinema amp/receiver and lights don't react to my audio, can work around by using mic but it's not ideal, i'd really like to be able to use the optical if possible

**Comment by psieg** [comment]
Is there no windows sound device for your optical output?

**Comment by sam-mckay** [comment]
Windows sound lists "Realtek Digital Output" for my optical connection, and Priskatik does show this as an option but it doesn't seem to pick up on any sound being played, the lights just don't react at all
![image](https://user-images.githubusercontent.com/9663483/32138695-31782ef8-bc2f-11e7-82e1-35ba0d76a4ef.png)

**Comment by psieg** [comment]
Does bar on the right next to Realtek Digital Output move when you play music?

**Comment by sam-mckay** [comment]
Yeah it does, i've managed to confirm that everything else is setup correctly by testing it with a HDMI connection as well. So it's just the digital output that isn't working

**Comment by psieg** [comment]
Prismatik uses a library called BASS to capture output. If you can, try [this](https://www.codeproject.com/Articles/797537/Making-an-Audio-Spectrum-analyzer-with-Bass-dll-Cs) project (download the zip [you need an account]) and build it yourself on run the exe included.
Does that program work (i.e. do the bars move)?

I don't think it will - whether that's a BASS or driver issue I don't know. Do other programs that capture you audio output work?
